### ISSUE TITLE
Howto start your demo

### ISSUE THREAD

**Issue by Phmichau** [fis-sst/BlazorMaps#5]
Hi

I try to start you demo but nothing

**Comment by mchilicki** [comment]
Could you tell us what is wrong for you, some error message or anything? Are you using Visual Studio 2019 with ASP.NET Core libraries installed?
Is newly created Blazor project is starting for you?

**Comment by ledpup** [comment]
The error message I get is:

`System.Text.Json.JsonException: ''<' is an invalid start of a value. Path: $ | LineNumber: 0 | BytePositionInLine: 0.'`

This is in Visual Studio 2022 Community Edition. Both the projects build fine.

Cryptic error message...

**Comment by petermarshall** [comment]
Thats an error message coming back from your api code rather than json.

*Peter Marshall*
07979608139
***@***.***
CV
<https://onedrive.live.com/view.aspx?resid=5099FD58F1491948%21574735&app=Word>
creator of https://nearby.cafe <http:%5C%5Cnearby.cafe>
https://petermarshall.github.io/blog/

**Comment by P33tr** [comment]
Make sure the api endpoint you are calling is working. That is an html error message comming back from a json endpoint.

**Comment by ledpup** [comment]
No, it's not my API. I'm just describing what I get when I run the **FisSst.BlazorMaps.Examples** project from VS 2022 after cloning from the repo. I haven't made a change to any of the code after pulling the repo. Just built and run the examples project.

In my own codebase, I have the BlazorMaps component working just fine. Got the map, markers, zoom, translation, etc. all working well.

I think there is something different in the examples project. I just tried update to .net 6, updated a few of the libraries in NuGet. Still no good.

Doesn't bother me too much. I can just read the examples anyway.
